.gitignore

```diff
@@ -13,6 +13,8 @@ BenchmarkDotNet.Artifacts/
 .gradle/
 src/SignalR/clients/**/dist/
 modules/
+dist/
```

Comment:
what are these?

Reply:
developing in the same repo as docs where these build output folders are ignored, this was expedient in order to work on both tasks